docs/conf.py

```diff
@@ -120,6 +120,14 @@
 
 # -- Options for HTML output -------------------------------------------
 
+# Set canonical URL from the Read the Docs Domain
+html_baseurl = os.environ.get("READTHEDOCS_CANONICAL_URL", "")
```

Comment:
> * Should I change this line to not use an empty string as the default, but instead show `image.dask.org`, like this?
>
> ```python
> html_baseurl = os.environ.get("READTHEDOCS_CANONICAL_URL", "image.dask.org")
> ```


After reading [this doc page]( https://docs.readthedocs.io/en/stable/canonical-urls.html ), it sounds like RTD checks the domains listed in the RTD settings, which is just `image.dask.org` (as seen below)

<img width="1840" alt="Screenshot 2024-09-24 at 9 35 39 PM" src="https://github.com/user-attachments/assets/a8b7baf1-3a5e-4059-a098-25037ec50934">

Then passes it in as the environment variable `READTHEDOCS_CANONICAL_URL`, which we pick up on the code line above

While we could change the fallback default, it is sounds like

1. Sounds like RTD does the right for us
2. Having a default that smokes out any issues (like empty string does) is useful
3. DRY

Think for these reasons having the setting in one place should work for us and keep maintenance straightforward

Though do appreciate this question and the exercise of thinking through it was helpful